app/views/local_transactions/_fields.html.erb

```diff
@@ -48,6 +48,41 @@
           <%= f.text_area :need_to_know, rows: 4, disabled: @resource.locked_for_edits?, class: "input-md-7 form-control" %>
         </span>
       </div>
+
+      <% administrations = {
+        "Scotland" => :scotland_availability,
+        "Wales" => :wales_availability,
+        "Northern Ireland" => :northern_ireland_availability
+      } %>
+      <% administrations.each do |title, field_name| %>
+        <%= f.fields_for field_name do |f| %>
+          <h3 class="govuk-heading-s"><%= f.label :type, title %></h3>
+          <div class="form-group">
```

Comment:
(trying to do this as a separate comment but github won't let me do that on the same line) 

This seems to be the only heading element in this template. I don't know the context - can you confirm that a `h3` here fits into the overall heading structure of the page?

Also just curious, what does it output as the content for this element? It looks like it might be a label element, which doesn't belong in a heading.

Reply:
Thanks Andy - fixed up